### ISSUE TITLE
Training 1024 * 1024 resolution images default setting

### ISSUE THREAD

**Issue by crleagure** [rosinality/stylegan2-pytorch#27]
Hi:
According to the generated sample,i think that the result is very good, so i want to train the 1024 * 1024 resolution images, could i know your default setting.Besides, there is another problem bother me is that how to evaluate the trained model, could you give me some suggestion. Thank you.

**Comment by rosinality** [comment]
You can use default arguments. (But you need to set batch size so that total batch size is 32.) And you can use FID scores for evaluate the model. (calc_inception.py for calculating inception statistics for real data, fid.py for calculating FID scores to compare generator statistics and real data statistics.)

**Comment by crleagure** [comment]
@rosinality Hi, I want to konw what function which FusedLeakyReLU 、fused_leaky_relu and upfirdn2d implements, and can i replace this with official implementation， for example，nn.LeakyReLU

**Comment by rosinality** [comment]
FusedLeakyReLU is adding bias + nn.LeakyReLU. Also you can use pytorch implementation of upfirdn2d https://github.com/rosinality/stylegan2-pytorch/blob/master/op/upfirdn2d.py#L152

**Comment by crleagure** [comment]
Hi，I have a question that the loss which calculated by F.softplus() can be used to back propagation

**Comment by rosinality** [comment]
Could you clarify the problem more?

**Comment by crleagure** [comment]
For example, The script train.py has a "d_logistic_loss", which calculate the descriminator loss by F.softplus, so I want to know that the loss which calculated by F.softplus  can be backpropagated

**Comment by rosinality** [comment]
Yes, of course.

**Comment by crleagure** [comment]
But, When use the F.softplus() to calculate the descriminator loss in other framework, I found that the network weight have not updated, do you have some suggestions?

**Comment by rosinality** [comment]
It is hard to guess without the code. You will need to check gradient propagation, but softplus itself will not be the problem.

**Comment by crleagure** [comment]
Thank you， I have solved this problem.
